client/dashboard/task-list/tasks/shipping/index.js

```diff
@@ -195,6 +201,11 @@ class Shipping extends Component {
 							recordEvent( 'tasklist_shipping_label_printing', { install: false } );
```

Comment:
We might want to update the two events here to pass which shipping extension was installed.

Reply:
👍 added.